### ISSUE TITLE
Unable to compile with FFmpeg 4.3

### ISSUE THREAD

**Issue by paddatrapper** [savonet/ocaml-ffmpeg#43]
Compiling with ffmpeg 4.3 results in the following error:

```
ocamlc.opt -thread -c -annot -bin-annot -g -w +A-4@5-7@8-9@11@12@20-35-44-45-50 avcodec.mli
File "avcodec.mli", line 149, characters 12-26:
149 |   type id = Codec_id.video
                  ^^^^^^^^^^^^^^
Error: Unbound type constructor Codec_id.video
make[4]: *** [OCamlMakefile:1048: avcodec.cmi] Error 2
```

Full bug report: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=963360

**Comment by paddatrapper** [comment]
Fixed in 0.4.2 release

**Comment by toots** [comment]
Yes! Forgot to close this one thks!
